tests/unit/test_upgrade.py

```diff
@@ -170,7 +168,6 @@ def test_on_upgrade_changed(harness):
         _member_started.return_value = True
         harness.charm.on.upgrade_relation_changed.emit(relation)
         _update_config.assert_called_once()
-        _send_signal.assert_called_once_with(SIGHUP, "postgresql")
```

Comment:
Why do you remove it?

Reply:
> Why do you remove it?

Because now the reload REST call in `update_config()` will work during the upgrade and we no longer do the additional SIGHUP.